app/models/embedded_ansible_worker/runner.rb

```diff
@@ -36,6 +38,17 @@ def setup_ansible
     _log.info("calling EmbeddedAnsible.start")
     EmbeddedAnsible.start
     _log.info("calling EmbeddedAnsible.start finished")
+
+    5.times do
```

Comment:
`EmbeddedAnsible.start` is synchronous so this felt unnecessary to me, but this solved the 502 errors I was getting when testing on an appliance.

We can also make the `WAIT_FOR_ANSIBLE_SLEEP` a worker setting, but I like to lean towards fewer knobs where I can :smile: